doc/user_doc/vic_installation/ts_firewall_error.md

```diff
@@ -0,0 +1,47 @@
+# VCH Deployment Fails with Firewall Validation Error #
+When you use `vic-machine create` to deploy a virtual container host, deployment fails because firewall port 2377 is not open on the target ESXi host or hosts.
+
+## Problem ##
+Deployment fails with a firewall error during the validation phase: 
+
+<pre>Firewall must permit 2377/tcp outbound to use VIC.</pre>
+
+## Cause ##
+ESXi hosts communicate with the virtual container hosts via port 2377. For deployment of a virtual container host to succeed, port 2377 must be open for outgoing connections on all all ESXi hosts before you run `vic-machine create`. Opening port 2377 for outgoing connections on ESXi hosts opens port 2377 for inbound connections on the virtual container hosts.
```

Comment:
Actually the container VMs communicate with the VCH through this port via serial over LAN. This requires this port to be open.